### DISCUSSION TITLE
SingboxServer Networking Guide

### DISCUSSION THREAD

**Discussion by GerryDazoo** [GerryDazoo/Slinger#127] [General]
They say a picture is worth a thousands words. Here's some pictures, with a few words.
I'll stick this in the repository after I get some feedback.

[SlingboxServerNetworkingGuide.pdf](https://github.com/GerryDazoo/Slinger/files/9759006/SlingboxServerNetworkingGuide.pdf)

**Comment by waldorf-statler** [comment]
Hi, this is great and will be using it this weekend to play with configurations, may have more comments after. For now a very minor editorial comment, the white labels in the diagrams are really hard to read, at least for me (basically anything apart from SS/MP/SB needs squinting!)

**Comment by bparatore** [comment]
Hi,
I got the server to run and connect to the slingbox, but I get this message: "Register: Error Getting Exteral IP Address, Retrying in 10 minutes" 
Any help appreciated. Thanks!

**Reply by GerryDazoo** [reply]
This has been resolved but it won't affect your ability to stream. 
See here. https://github.com/GerryDazoo/Slinger/discussions/236.
Unconfigure your finderid if you don't need it. Please read the release notes.

**Reply by bparatore** [reply]
Thanks!
